source/draw.h

```diff
@@ -61,4 +64,6 @@ extern int font_kerningamount[96];
 qpic_t *Draw_CachePic (char *path);
 #ifdef __PSP__
 qpic_t *Draw_CacheImg (char *path);
-#endif // __PSP__
\ No newline at end of file
+#elif __WII__
+qpic_t *Draw_LMP (char *path);
```

Comment:
Can this be removed?

Reply:
I would prefer to keep LMP and CachePic logic separate on this platform. Whenever these functions are consolidated it creates a small lag when drawing Textboxes. This is because there are 5 file open/closes for 10 total textures loaded in coupled with slow IO speeds

Reply:
I'm not sure I understand. We shouldn't be using lmp anywhere.

Reply:
lmp is used to draw the console keyboard background.

Reply:
I see. We should depreciate this in favor of drawing it in engine.